evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -188,6 +188,23 @@
             <a href="{% url 'staff:evaluation_copy' evaluation.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" title="{% translate "Copy" %}">
                 <span class="fas fa-copy"></span>
             </a>
+            {% if not evaluation.has_exam %}
+            <confirmation-modal type="submit" id="exam_creation_modal" name="evaluation_id" value="{{ evaluation.id }}" confirm-button-class="btn-primary" form="exam_creation_form_{{ evaluation.id }}">
+                <span slot="title">{% translate 'Create exam evaluation' %}</span>
+                <span slot="action-text">{% translate 'Create exam evaluation' %}</span>
+                <span slot="question">
+                    {% blocktranslate %}
+                    Create an exam evaluation based on this evaluation. This will copy all the participants and contributors from the original evaluation. It will set the weight of the original evaluation to 9 and its end date will be set to the day before the exam.
+                    {% endblocktranslate %}
+                </span>
+                <span slot="extra-inputs">
+                    <label for="exam_date">{% translate 'Exam Date:' %}</label>
+                    <input type="date" id="exam_date" name="exam_date" required form="exam_creation_form_{{ evaluation.id }}">
```

Comment:
This id will also be duplicated. We could include the evaluation's id, or it should also work to move the `input` element into the `label` element, then we don't need the `id` and `for` attributes (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). I think I prefer the second approach, we would have to make sure it also works with `type="date"` though :)